app/helpers/react_components/student/solution_summary.rb

```diff
@@ -0,0 +1,25 @@
+module ReactComponents
+  module Student
+    class SolutionSummary < ReactComponent
```

Comment:
@iHiD I don't know what to name this component.

Reply:
Yeah - I struggled with this too. This is fine for now - I want us to do a big renaming refactor before launch anyway, so let's leave it until then.